src/Microsoft.Azure.Functions.PowerShell.SDK.csproj

```diff
@@ -6,12 +6,22 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="test\**" />
+    <EmbeddedResource Remove="test\**" />
+    <None Remove="test\**" />
+  </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.2.1" />
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="test\E2E\apps\empty-app\" />
+    <Compile Update="resources\AzPowerShellSdkStrings.Designer.cs">
```

Comment:
What is this file for?

Reply:
Visual Studio keeps autogenerating it. I have removed it several times but it keeps coming back, so I figured I'd just leave it. It's not harmful to the resource generation

Reply:
Ok, so let's keep it as is for now.